packages/node/example/demoClientApp/src/index.js

```diff
@@ -112,8 +114,17 @@ app.get("/redirect", async (req, res) => {
           loginStatus = `Received another redirect, but we are already handling a previous redirect request - so ignoring this one!`;
           sendHtmlResponse(res);
         } else if (info.isLoggedIn) {
-          webIdToRead = info.webId;
+
+          // This is deliberately making a big assumption about the structure of
+          // the logged-in WebID, but if the WebID does have a different
+          // structure, which is perfectly valid of course, then this code
+          // shouldn't alter it, and the user is free to enter whatever resource
+          // URI they actually want anyway, so it's just a convenience in the
+          // 'common' case.
+          resourceToRead = info.webId.replace("/profile/card#me", "/private/");
```

Comment:
Same here, I get the intent, but is it something that we want to feature in a demo app that we advertize for if it's a pattern we discourage ?

Reply:
Yeah, again this was just a very convenient, hacky way to quickly demo accessing a private resource - it's definitely not ideal (hence the comment). We could consider using the previous-spec-mandated Private Type Registry instead, but I'm not sure what the current spec says about that (if anything at all). But yeah, if there is a spec-mandated way to safely determine a known private resource in all Pods, then this code should be updated to use that instead.